static/js/controllers/edit-contact.js

```diff
@@ -17,7 +17,7 @@ var libphonenumber = require('libphonenumber/utils'),
           endkey: [{}],
           reduce: false,
           include_docs: true,
-          timeout: false
+          targetScope: "contacts"
```

Comment:
But if state is changed, do we care about the old request for contents for this dialog?